targets/lantiq-xrx200

```diff
@@ -10,3 +10,17 @@ device('avm-fritz-box-7362-sl', 'avm_fritz7362sl', {
 device('avm-fritz-box-7412', 'avm_fritz7412', {
         factory = false,
 })
+
+-- TP-Link
+
+  -- CAVEAT: These devices don't have a dedicated WAN port.
+  --         All ethernet ports on the device are bridged and
+  --         used as WAN ports. Clients connected to these
+  --         ports will be connected to your private network.
```

Comment:
Meh, I think this isn't user-visible enough and we should rather make that a note in the supported device list. Sorry this is coming so late.

Reply:
We already have the `lan_as_wan` note, so I'm going to add that.

Reply:
thanks